Crash: AssertionError: Unknown preview type: ERROR

Reported via Google Play for 5.200-RC1

```
java.lang.AssertionError: Unknown preview type: ERROR
	at com.fsck.k9.fragment.MessageListFragment$MessageListAdapter.getPreview(MessageListFragment.java:1968)
	at com.fsck.k9.fragment.MessageListFragment$MessageListAdapter.bindView(MessageListFragment.java:1886)
	at android.widget.CursorAdapter.getView(CursorAdapter.java:289)
	at android.widget.HeaderViewListAdapter.getView(HeaderViewListAdapter.java:220)
	at android.widget.AbsListView.obtainView(AbsListView.java:2363)
	at android.widget.ListView.makeAndAddView(ListView.java:1970)
	at android.widget.ListView.fillDown(ListView.java:704)
	at android.widget.ListView.fillFromTop(ListView.java:765)
	at android.widget.ListView.layoutChildren(ListView.java:1716)
	at android.widget.AbsListView.onLayout(AbsListView.java:2162)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1741)
	at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1585)
	at android.widget.LinearLayout.onLayout(LinearLayout.java:1494)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at com.android.internal.widget.ActionBarOverlayLayout.onLayout(ActionBarOverlayLayout.java:493)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
	at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
	at com.android.internal.policy.DecorView.onLayout(DecorView.java:726)
	at android.view.View.layout(View.java:17639)
	at android.view.ViewGroup.layout(ViewGroup.java:5612)
	at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2341)
	at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2068)
	at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1254)
	at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:6300)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:874)
	at android.view.Choreographer.doCallbacks(Choreographer.java:686)
	at android.view.Choreographer.doFrame(Choreographer.java:621)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:860)
	at android.os.Handler.handleCallback(Handler.java:751)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6088)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```

--- a/k9mail/src/main/java/com/fsck/k9/fragment/MessageListFragment.java
+++ b/k9mail/src/main/java/com/fsck/k9/fragment/MessageListFragment.java
@@ -1954,7 +1954,8 @@ public class MessageListFragment extends Fragment implements OnItemClickListener
             DatabasePreviewType previewType = DatabasePreviewType.fromDatabaseValue(previewTypeString);
 
             switch (previewType) {
-                case NONE: {
+                case NONE:
+                case ERROR: {
                     return "";
                 }
                 case ENCRYPTED: {

